src/EntityFrameworkMock.Moq/EntityFrameworkMock.Moq.csproj

```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Castle.Core" Version="[4.3.1, 5.0.0)" />
-    <PackageReference Include="Moq" Version="[4.10.0, 5.0.0)" />
+    <PackageReference Include="Castle.Core" Version="4.4.1" />
```

Comment:
Can we keep the version range, but just change the minimum version? So this version would become: `"[4.4.1, 5.0.0)"`.